crates/ubrn_cli/src/android.rs

```diff
@@ -232,12 +232,12 @@ impl AndroidArgs {
             .arg(target.to_string())
             .arg("--platform")
             .arg(format!("{}", api_level));
-        if !self.common_args.release {
+        if self.common_args.profile() != "release" {
             cmd.arg("--no-strip");
         }
         cmd.arg("--").arg("build");
-        if self.common_args.release {
-            cmd.arg("--release");
+        if self.common_args.profile() != "debug" {
+            cmd.arg("--profile").arg(self.common_args.profile());
```

Comment:
Nit: use local variable
```suggestion
        if profile != "debug" {
            cmd.args(["--profile", profile]);
```